pupil_src/shared_modules/glfw.py

```diff
@@ -656,6 +657,14 @@ def glfwGetMonitorPos(monitor):
     return xpos.value, ypos.value
 
 
+def glfwGetMonitorWorkarea(monitor):
+    xpos, ypos, width, height = c_int(0), c_int(0), c_int(0), c_int(0)
+    _glfw.glfwGetMonitorWorkarea(
+        monitor, byref(xpos), byref(ypos), byref(width), byref(height)
+    )
+    return xpos.value, ypos.value, width.value, height.value
```

Comment:
I think returning a named tuple would make sense here, e.g. `Rect` (similar to `NSRect` in ObjC)